Sources/SnapshotPreviewsCore/ExpandingViewController.swift

```diff
@@ -92,5 +118,42 @@ public final class ExpandingViewController: UIHostingController<EmergeModifierVi
     }
   }
 
+//  MARK: - Timer
+
+  func startTimer() {
+      if self.timer == nil {
+          self.startTime = Date()
```

Comment:
Can we eliminate all the `self.` that aren't necessary? I noticed they aren't used in other functions like "stopAndResetTimer", we shouldn't have them here either